nftables.go

```diff
@@ -105,7 +105,7 @@ func newInternal(family Family, table string, execer execer) (Interface, error)
 	// Check that (a) nft works, (b) we have permission, (c) the kernel is new enough
 	// to support object comments.
 	cmd = exec.Command(nft.path, "--check", "add", "table", string(nft.family), nft.table,
-		"{", "comment", `"test"`, "}",
+		"{", "comment", `"test"`, ";", "}",
```

Comment:
:man_facepalming: 

We should rewrite this to use `nft.Check` now that that exists. Then that gets rid of the opportunity to get the syntax wrong...

Reply:
done